### PR TITLE
Created models for posts and base apps

### PR DESCRIPTION
**Pull request by jiteshm17**
Please let me know any corrections to be made.

### CODE REVIEW

File: posts/models.py [hunk 1]
```diff
@@ -1,3 +1,42 @@
...
+from django.contrib.auth.models import User
+from base.models import Club
+
 
 # Create your models here.
+
+class Post(models.Model):
+    post_creator = models.ForeignKey(User, on_delete=models.CASCADE)
+    club = models.ForeignKey(Club, on_delete=models.CASCADE)
+    is_approved = models.BooleanField()
```
Comment (adwait-thattey) [inline posts/models.py]: put this as default false

File: posts/models.py [hunk 2]
```diff
@@ -1,3 +1,42 @@
...
+    description = models.TextField(max_length=150)
```
Comment (adwait-thattey) [inline posts/models.py]: This is a rich text field that will come from ck editor. Ask ankur to do it.

File: posts/models.py [hunk 3]
```diff
@@ -1,3 +1,42 @@
...
+    video = models.URLField()
+
+
+class Image(models.Model):
+    post = models.ForeignKey(Post, on_delete=models.CASCADE)
+    image_url = models.URLField()
```
Comment (adwait-thattey) [inline posts/models.py]: 1. directly use image=ImageField()
2. The is_public option is in post. not in image

File: posts/models.py [hunk 4]
```diff
@@ -1,3 +1,42 @@
...
+    is_public = models.BooleanField(default=False)
+
+
+class Poll(models.Model):
+    post = models.OneToOneField(Post, on_delete=models.CASCADE)
+
+
+class Option(models.Model):
+    poll = models.ForeignKey(Poll, on_delete=models.CASCADE)
+    options = models.CharField(max_length=150)
```
Comment (adwait-thattey) [inline posts/models.py]:
```suggestion
    option = models.CharField(max_length=150)
```

File: posts/models.py (hunk 3 above)
Comment (adwait-thattey) [reply posts/models.py]: yes

**Comment by adwait-thattey** [comment]
The build is still failing. Please check the issue and resolve. I think it's because pillow is not in requirements.txt

U can update requirements.txt by running

`pip3 freeze>requirements.txt` from the virtual environment in pycharm

**Comment by jiteshm17** [comment]
The build is failing because pillow library has not been installed for the imagefield()

**Comment by jiteshm17** [comment]
I realized the mistake. Give me a moment to rectify it

**Pull request merged**